src/component/options.ts

```diff
@@ -24,17 +26,22 @@ export interface GlobalOptions {
   navigation?: NavigationConfiguration;
 }
 
+/**
+ * @category Initialization
+ */
 export interface NavigationConfiguration {
   /**
    * @default false
    */
   enabled: boolean;
```

Comment:
this one is not mandary

Reply:
The **navigation** field in **GlobalOptions** is already optional.
If the user instantiate this object, the **enabled** field should be mandatory, else I don't see the point to instantiate this object (or we can not have this field).

Reply:
good point, so let's keep it as it is.